docs/env.md

```diff
@@ -266,3 +269,24 @@ Example: `https://example.com/api/update-state`.
 - Type: string
 - Required: Yes
 - Default: None
+
+## 3. Environment to create the state bucket in AWS
+
+Specified in `{REPO_ROOT}/src/aws/env.d/state_bucket`.
+
+#### AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY
+
+A key ID + secret pair for an AWS IAM account with administrative access to the resources we need to access to create the state bucket (see Terraform configuration files).
+
+- Type: string
+- Required: Yes
+- Default: None
+
+#### TF_VAR_aws_region
```

Comment:
in `state_bucket.dist` file this variable isn't prefixed with `TF_VAR_`

Reply:
Fixed it.